boards/arm/adafruit_trinket_m0/doc/adafruit_trinket_m0.rst

```diff
@@ -0,0 +1,147 @@
+.. _adafruit_trinket_m0:
+
+Adafruit Tinket M0
+##################
+
+Overview
+********
+
+The Adafruit Trinket M0 is a tiny (27 mm x 15 mm) development board
+with an onboard RGB LED, USB port, and range of I/O broken out onto 5
+pins.
+
+.. image:: img/adafruit_trinket_m0.png
+     :width: 500px
+     :align: center
+     :alt: Adafruit Trinket M0
+
+Hardware
+********
+
+- ATSAMD21E18A ARM Cortex-M0+ processor at 48 MHz
+- 256 KiB flash memory and 32 KiB of RAM
+- Internal trimmed 8 MHz oscillator
+- A user LED
+- A RGB DotStar LED
+- Native USB port
+- One reset button
+
+Supported Features
+==================
+
+The adafruit_trinket_m0 board configuration supports the following hardware
+features:
+
++-----------+------------+--------------------------------------+
+| Interface | Controller | Driver/Component                     |
++===========+============+======================================+
+| NVIC      | on-chip    | nested vector interrupt controller   |
++-----------+------------+--------------------------------------+
+| Flash     | on-chip    | Can be used with NFFS to store files |
++-----------+------------+--------------------------------------+
+| SYSTICK   | on-chip    | systick                              |
++-----------+------------+--------------------------------------+
+| WDT       | on-chip    | Watchdog                             |
++-----------+------------+--------------------------------------+
+| GPIO      | on-chip    | I/O ports                            |
++-----------+------------+--------------------------------------+
+| USART     | on-chip    | Serial ports                         |
++-----------+------------+--------------------------------------+
+| SPI       | on-chip    | Serial Peripheral Interface ports    |
++-----------+------------+--------------------------------------+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the Kconfig
+:file:`boards/arm/adafruit_trinket_m0/adafruit_trinket_m0_defconfig`.
+
+Connections and IOs
+===================
+
+The `Adafruit Learn`_ site has detailed information about the board
```

Comment:
More descriptive if you write this as (and adjust the label name at the end of the doc): 

    The `Adafruit Trinket M0 Learn site`_ has ...

Reply:
Done